docs/modules/user-guides/pages/appium-setup.adoc

```diff
@@ -0,0 +1,427 @@
+[source,shell]
+----
+npm install -g appium
+----
+To run Appium use the command:
+[source,shell]
+----
+appium
+----
+====
+
+== Basic setup
+
+Make sure you have installed:
+
+- https://www.java.com/en/download/manual.jsp[Java]
+- http://developer.android.com/sdk/index.html[Android SDK]
+- https://developer.apple.com/xcode/[Xcode (iOS)]
+
+Set environment variables:
+
+- JAVA_HOME
+- ANDROID_HOME
+
+TIP: You can open "Edit configuration" to set the variables in Appium Desktop.
+
+Read more details for basic http://appium.io/docs/en/drivers/android-uiautomator2/[Android] and http://appium.io/docs/en/drivers/ios-xcuitest/[iOS] setup in Appium Guide.
+
+== Appium-Doctor (Optional)
+
+
+https://github.com/appium/appium-doctor[Appium-Doctor] diagnoses, reports and fixes common Node, iOS and Android configuration issues before starting Appium. It helps to make sure you have everything to start using Appium.
+
+Appium-doctor checks for:
+
+* Android:
+** Android sdk exists and configured properly;
+** Environment variables and path check;
+* iOS:
+** XCode is installed (with command line tools);
+** Dev tools security check;
+** Auth check;
+** Node binary check;
+
+== Appium Capabilities
+
+Desired Capabilities are just a set of keys and values that get sent to the Appium server during session initialization, that tell Appium what kind of thing we want to automate. The minimum set of required capabilities for any Appium driver should include:
+
+- *platformName*: the name of the platform to automate
+- *platformVersion*: the version of the platform to automate
+- *deviceName*: the kind of device to automate
+- *app*: the path to the app you want to automate (but use the browserName capability instead in the case of automating a web browser)
+- *automationName*: the name of the driver you wish to use
+
+There are lots of other capabilities. To find out more please see http://appium.io/docs/en/writing-running-appium/caps/index.html[Appium Capabilities Doc].
+
+=== Setting capabilities in VIVIDUS
+
+Before running VIVIDUS scenarios it is necessary to pass Desired Capabilities to Appium Server. Use them as properties with `selenium.grid.capabilities.` prefix. Then just put these capabilities into the appropriate `*.properties` file. Please see examples below:
+
+==== Basic properties
+
+[source, property]
+----
+# Example of iOS properties
+selenium.grid.capabilities.platformName=iOS
+selenium.grid.capabilities.platformVersion=15.5
+selenium.grid.capabilities.deviceName=iPad Pro (9.7-inch)
+selenium.grid.capabilities.automationName=XCUITest
+selenium.grid.capabilities.app=/Users/myDirectory/application.ipa
+
+# Example of Android properties
+selenium.grid.capabilities.platformName=android
+selenium.grid.capabilities.platformVersion=11.0
+selenium.grid.capabilities.deviceName=Pixel 6
+selenium.grid.capabilities.automationName=UIAutomator2
+selenium.grid.capabilities.app=/Users/myDirectory/application.apk
+----
```

Comment:
Imho it's better to put each set of props into separate  source section e.g.
.Example of iOS properties
[source, property]
----
......
----

.Example of Android properties
[source, property]
----
......
----